src/css/angular-tooltips.css

```diff
@@ -36,44 +36,87 @@
   transition: opacity 0.2s linear;
   opacity: 1;
 }
+._720kb-tooltip-caret:before {
+  content:'';
+  position: absolute;
+  width: 0;
+  height: 0;
+  border: 6px solid rgba(0, 0, 0, 0.8);
+}
 ._720kb-tooltip-caret:after {
   content:'';
   position: absolute;
   width: 0;
   height: 0;
+  border: 5px solid rgba(0, 0, 0, 0.8);
 }
-._720kb-tooltip-left ._720kb-tooltip-caret:after {
+._720kb-tooltip-left ._720kb-tooltip-caret:before {
   top: 50%;
   left: 100%;
   margin-left:0;
   margin-top: -6px;
-  border-left: 6px solid rgba(0, 0, 0, 0.8);
-  border-top: 6px solid transparent;
-  border-bottom: 6px solid transparent;
+  border-top-color: transparent !important;
```

Comment:
@45kb I use !important because it's really important to create triangles. :) 
if remove it, the color I set in my custom css class will overwrite it, which is not what I want.
In my custom css class, I just need to set the border color(the "before" border color) and the content color(the "after" color). I think it's really convenient for others to use tooltips.
![image](https://cloud.githubusercontent.com/assets/2586026/5787773/952e6d1a-9e49-11e4-9741-87ed2fb7201b.png)